packages/vike-vue/src/hooks/useConfig/useConfig-server.ts

```diff
@@ -45,7 +45,13 @@ function setConfigOverPageContext(config: ConfigFromHook, pageContext: PageConte
     const configValue = config[configName]
     if (!configValue) return
     pageContext._configFromHook![configName] ??= []
-    pageContext._configFromHook![configName].push(configValue)
+
```

Comment:
Both Brillout and I checked and it's good as it is now - in the end the lists is flattened

Reply:
Thanks, I’ve reverted the changes.